### DISCUSSION TITLE
Consiglio cronotermostato per casa nuova

### DISCUSSION THREAD

**Discussion by Eddy16391** [indomus/forum#1541] [Hardware & Firmware]
Ciao ragazzi,
sto per iniziare degli importanti lavori di ristrutturazione nella mia abitazione e volevo rendere il tutto più connesso possibile.
Visto che andrò ad installare un impianto di riscaldamento a pavimento volevo mettere un termostato che era in grado di gestire le differenti zone della casa tramite il collegamento dei sensori di temperatura che già usavo, cioè quelli della Sonoff.
Mi spego meglio, sono indeciso se prendere un termostato fisico (se esiste) oppure programmarmelo in Home Assistant, perchè volevo una cosa del tipo, imposto la temperatura target di casa a 22°, ma se una stanza arriva a quella temperatura prima delle altre instanto spengo i riscaldamenti in quella stanza.
Non so se mi sono spiegato bene. Grazie per eventuali consigli :-D

**Comment by vinny74man** [comment]
Di solito però un impianto multizona non funziona così, imposti la temperatura nella singola stanza e quando arriva quella stanza a temperatura si chiude l'afflusso di acqua calda o fredda a seconda della stagione in quella stanza tramite valvole, quindi avendo già da quello che ho capito tu dei sonoff che immagino abbiano anche il contatto per aprire e chiudere le valvole dovresti creare solo dei termostati generici per stanza che leggono dal sensore e comandano la valvola, poi se vuoi un setpoint generale potresti fare un sensore template che fa la media dei sensori e nel caso apre o chiude tutte le valvole. Penso che questa sia la soluzione da intraprendere.
